src/gameplaykit.cs

```diff
@@ -48,7 +77,7 @@ interface GKAgent {
 		float Radius { get; set; }
 
 		[Export ("speed")]
-		float Speed { get; }
+		float Speed { get; [iOS (10,0), TV(10,0), Mac (10,12, onlyOn64: true)] set; }
```

Comment:
I'm fairly sure we did the same several times in the past (not that it means it's correct) and it does work (using a binary .dll that only overrides the get) on the simulator and device.